apm-agent-core/src/test/java/co/elastic/apm/agent/impl/ElasticApmTracerTest.java

```diff
@@ -174,6 +179,44 @@ void testRecordException() {
         assertThat(error.getTransactionInfo().isSampled()).isFalse();
     }
 
+    @Test
+    void testDoesNotIgnoreExceptionsWithEmptyWildcards(){
```

Comment:
Basically, this is the same as `testRecordException`, right?

Reply:
test removed to avoid duplication